azure-resources/ContainerService/managedClusters/recommendations.yaml

```diff
@@ -27,7 +27,7 @@
   recommendationResourceType: Microsoft.ContainerService/managedClusters
   recommendationMetadataState: Active
   longDescription: |
-    AKS assigns the kubernetes.azure.com/mode: system label to nodes in system node pools signaling system pods should be scheduled there.
+    AKS assigns the kubernetes.azure.com/mode: system label to nodes in system node pools signaling only system pods should be scheduled there. For worker nodes assign taints and label CriticalAddonsOnly to node.
```

Comment:
```suggestion
    AKS assigns the kubernetes.azure.com/mode: system label to nodes in system node pools signaling the preference for system pods should be scheduled there. The CriticalAddonsOnly=true:NoSchedule taint can be added to your system nodes to enforce the prevention of application pods from being scheduled on them. 
```

Reply:
Adjust the wording as the system label doesn't explicitly block application pods from being added to those system nodes. Also adding clarity to use same terminology referenced in docs around system nodes.

Reply:
Good with it the new wording and labeling :) LTGM

Reply:
I have approved the commit but it seems that the number of character is more than 300. Can you please readjust few characters in your suggestion to make it comply with the description.